src/main/webapp/js/sdk_release.js

```diff
@@ -0,0 +1,29 @@
+async function getSDKRelease(platform, releaseName, libraryName) {
+  const response = await fetch(`v1/platforms/${platform}/releases/${releaseName}/sdks/${libraryName}`);
+  const sdkReleaseMetadata = await response.json();
+
+  console.log(sdkReleaseMetadata);
+  for (const [key, value] of Object.entries(sdkReleaseMetadata)) {
+    if (key === "platform" || key === "libraryName") {
+      continue;
+    } else if (key === "additionalInfo") {
```

Comment:
What values does the additional info key have?

Reply:
additional info will have all the optional fields such as ariane entry, release note cl, javadocs cl, link to changelog, internal hotlist, prebuilts drop bug, manual verification bug, javadoc bug